src/auth/dtos/auth-register-output.dto.ts

```diff
@@ -19,4 +19,12 @@ export class RegisterOutput {
   @Expose()
   @ApiProperty({ example: [ROLE.USER] })
   roles: ROLE[];
+
+  @Expose()
+  @ApiProperty()
+  email: string;
+
+  @Expose()
```

Comment:
Was wondering if we need this to be exposed in register output?

Reply:
@Saad-Amjad 
Nice observation! 
I didn't think that far. 
But I'll still leave it in for now. We can remove it after discussing it with other members as well. :)